internal/server/data/passwordresettoken_test.go

```diff
@@ -0,0 +1,23 @@
+package data
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+
+	"github.com/infrahq/infra/internal/server/models"
+	"github.com/infrahq/infra/uid"
+)
+
+func TestRemoveExpiredPasswordResetTokens(t *testing.T) {
+	tx := setupDB(t)
+	prt, err := CreatePasswordResetToken(tx, &models.Identity{Model: models.Model{ID: uid.New()}}, 0)
+	assert.NilError(t, err)
```

Comment:
Not a blocker, but I've been trying to create records that won't be deleted , to check that the filter behaves properly and leaves non-expired records. I think that'd be a good improvement to this test.